guide/getting-started/developers.md

```diff
@@ -104,7 +107,7 @@ A list of the available colours and their use cases can be found [in the Design
 ```scss
 // BannerText.scss
 
-@import '~@telusdigital/tds/dist/scss/colours';
+@import '~@tds/core-colours/colours';
```

Comment:
Realized we should add that the Colours are now a separate package and you will need to install it as a dependency. The getting started guide should call that out, and include a yarn install line for installing it, otherwise you won't be able to to the `@import`!